test/acceptance/editors-integration-test.js

```diff
@@ -66,4 +66,362 @@ describe('editors integration', function () {
       expect(template).toBe('<button type="button"></button>');
     });
   });
+
+  describe('with embedded templates', function () {
+    describe('hbs (tests and manual low-level usage)', function () {
```

Comment:
for other reviewers, even though this isn't related to `<template>`, this way of creating components (hbs + setComponentTemplate) works as early as 3.13 (iirc?), and has not been able to be linted _until now_. Additionally, folks could choose to have a different set of lints for their tests' templates, if they want.